src/components/LandingPage/LandingPage.scss

```
@@ -0,0 +1,80 @@
+@use "../../spacing" as *;
+@use "../../font-size" as *;
+@import "../../colours.scss";
+@import "../../mixins.scss";
+
+.landing-page-wrapper {
+  margin: 0 auto;
+  padding-bottom: $space-1;
+  text-align: center;
+}
```

whole wrapper needs to be centred vertically, plenty of ways but I use the lazy flexbox route 😆 

```suggestion
.landing-page-wrapper {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
  height: 100%;
  margin: 0 auto;
  padding-bottom: $space-1;
  text-align: center;
}
```